pages/doc/integrations_tas_howto.md

```diff
@@ -346,6 +346,65 @@ Log in to your Wavefront instance (for example, `https://example.wavefront.com`)
 3. Explore one or two dashboards. [Examine Data with Dashboards and Charts](ui_examine_data.html) has an overview and includes a video.
 4. As appropriate, clone any of the existing dashboards to add charts, modify queries, and more. See [Create, Customize, and Optimize Dashboards](ui_dashboards.html) and [Create and Customize Charts](ui_charts.html).
 
+### Step 5 (Optional): Use the Service Broker to Send Custom Application Metrics to the Wavefront Proxy
+
+1. In Ops Manager, in the **Wavefront Proxy Config** panel,
+   select the **Enable Service Broker Bindings** check box.
+2. To provision a Wavefront proxy service instance,
+   run the command:
+   `cf create-service wavefront-proxy standard SERVICE_INSTANCE_NAME`
+3. To bind apps to the service instance, run the command:
+   `cf bind-service APP_NAME SERVICE_INSTANCE_NAME`
+4. If you already have an existing app and want to update its service instance reference from the v3 broker,
+   first unbind it with this command:
+   `cf unbind-service APP_NAME SERVICE_INSTANCE_NAME`
+
+Note: For information about how apps can send metrics to the Wavefront proxy, see the [cloud-foundry-sampleapp](https://github.com/wavefrontHQ/cloud-foundry-sampleapp).
+
+### Step 5 (Optional): Connect Multiple Proxy Instances to an IAAS Load Balancer
+
+### Set up the load balancer in your IAAS
+
+If you use multiple Wavefront VMs,
+you will want to use your IAAS of choice to set up a load balancer.
+BOSH DNS does not provide load balancing,
+so increasing the VM count will result in one VM taking all the requests.
+
+In-depth instructions on configuring load balancers in different IAASes is beyond the scope of this documentation.
+You can see a detailed example across two sections of the TAS for VMs documentation:
+1. Initial setup of load balancers is required for
+   [AWS](https://docs.pivotal.io/ops-manager/3-0/aws/prepare-env-manual.html) and
+   [GCP](https://docs.pivotal.io/ops-manager/3-0/gcp/prepare-env-manual.html#loadbalancer),
+   but not for Azure.
+2. After the initial setup, you can use
+   [this](https://docs.pivotal.io/application-service/3-0/operating/configure-lb.html)
+   document for finalizing the configuration of the load balancers in all IAAS.
+
+You should adjust the naming in these docs to suit a Wavefront proxy.
+You can ignore the portions of these docs that contain information about interactions with Ops Manager, because since we will talk about that in the next section.
```

Comment:
My bad, "since" should be removed from the last portion of the sentence:

 .., because we will talk about that in the next section.